<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTPserverWindow</class>
 <widget class="QMainWindow" name="FTPserverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QToolButton" name="clearToolButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../application.qrc">
        <normaloff>:/images/cleanup.png</normaloff>:/images/cleanup.png</iconset>
      </property>
     </widget>
    </item>
    <item row="11" column="0" rowspan="2" colspan="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../application.qrc"/>
  <include location="../application.qrc"/>
  <include location="../application.qrc"/>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
